package.json

```diff
@@ -4,14 +4,14 @@
   "description": "Making company travel and accomodation easy and convinient",
   "main": "index.js",
   "scripts": {
-    "test": "npm run migration:refresh:test && NODE_ENV=test npm run db:seed && NODE_ENV=test nyc --reporter=html --reporter=text mocha --timeout 20000 -r @babel/register ./src/modules/**/*.test.js --exit",
+    "test": "npm run migration:refresh:test && cross-env NODE_ENV=test npm run db:seed && cross-env NODE_ENV=test nyc --reporter=html --reporter=text mocha --timeout 20000 -r @babel/register ./src/modules/**/*.test.js --exit cross-env NODE_ENV=test",
```

Comment:
@SteveFunso Kindly add `cross-env NODE_ENV=test ` before `npm run migration:refresh:test`
and remove the `cross-env NODE_ENV=test ` after `--exit`

Reply:
ok...  noted